### ISSUE TITLE
-1XXX has incorrect min / max

### ISSUE THREAD

**Issue by mixmix** [inukshuk/edtf.js#25]
```js
var test = '-1XXX'
var d = edtf(test)

console.log(new Date(d.min))
// -001000-01-01T00:00:00.000Z  << expect -1999 ? 
console.log(new Date(d.max))
// -001000-12-31T23:59:59.999Z << correct ?
```

I think this is a bug.

_Thanks for making this module, I'm using it to help create indigenous genealogy + history recording in New Zealand. Being able to record dates as humans have remembered them is a total gift_ :heart:

**Comment by barnaclebarnes** [comment]
@mixmix We're based in Auckland and using this in our [Curtis](https://www.curtisapp.com) platform so in a similar #glamtech space. What are you working on? ping me a note (barnaclebarnes pretty much anywhere, email at bottom of Curtis website).

**Comment by inukshuk** [comment]
@mixmix thanks! This should be fixed in 2.7.2.
